### PR TITLE
Controller UT

### PR DESCRIPTION
**Pull request by selim5**
#9 Controller UT

### CODE REVIEW

**Comment by selim5** [comment]
Error Im getting when running test
image

File: pkg/reconciler/mongodb/controller_test.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	"knative.dev/pkg/configmap"
+	logtesting "knative.dev/pkg/logging/testing"
+	. "knative.dev/pkg/reconciler/testing"
+
+	// Fake injection informers
+	_ "github.com/googleinterns/knative-source-mongodb/pkg/client/clientset/versioned/typed/sources/v1alpha1/fake"
+	_ "github.com/googleinterns/knative-source-mongodb/pkg/client/injection/client/fake"
+	_ "github.com/googleinterns/knative-source-mongodb/pkg/client/injection/informers/sources/v1alpha1/mongodbsource/fake"
+	_ "github.com/googleinterns/knative-source-mongodb/pkg/reconciler/testing"
+	_ "knative.dev/pkg/client/injection/kube/informers/batch/v1/job/fake"
```
Comment (nachocano) [inline pkg/reconciler/mongodb/controller_test.go:30]: do you need this job one?
I think you need the deployment informer fake
try adding this import, and remove the back job one...
_ "knative.dev/pkg/client/injection/kube/informers/apps/v1/deployment/fake"

File: pkg/reconciler/mongodb/controller_test.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+	_ "knative.dev/pkg/client/injection/kube/informers/core/v1/serviceaccount/fake"
```
Comment (nachocano) [inline pkg/reconciler/mongodb/controller_test.go:31]: same with this one... do you need the serviceAccount informer?

Comment (nachocano) [reply pkg/reconciler/mongodb/controller_test.go:31]: you will probably need the secret fake informer

Comment (nachocano) [reply pkg/reconciler/mongodb/controller_test.go:31]: _ "knative.dev/pkg/client/injection/kube/informers/core/v1/secret/fake"

**Pull request merged**